tests/runtest.proj

```diff
@@ -160,9 +161,11 @@ $(_XunitEpilog)
       File="$(XunitWrapperSrcDir)\project.json"
       Lines="$(XunitWrapperGenPackConfig)"
       Overwrite="true" />
-
-    <MSBuild Projects="$(XunitWrapperSrcDir)\$(XunitWrapper).csproj" />
-
+  </Target>
+  
+  <Target Name="BuildXunitWrapper">
+    <MSBuild Projects="$(XunitWrapperSrcDir)\$(XunitWrapper).csproj"
+             Properties="RestorePackages=false"/>
```

Comment:
Sure: it's good to capture as many project.jsons in a single `dotnet restore` call as possible, to avoid loading dotnet/nuget once per project and to allow restoring the project.jsons in parallel. So, I disabled the per-project "msbuild-batched" restore and started calling restore on the root dir of the projects.

(See "Restore all wrapper projects' project.jsons in one pass for perf & to avoid concurrency problems" below.)